apps/challenges/admin.py

```diff
@@ -44,24 +44,25 @@ class LeaderboardAdmin(ImportExportTimeStampedAdmin):
 class ChallengePhaseSplitAdmin(ImportExportTimeStampedAdmin):
     list_display = ("id", "challenge_phase", "dataset_split", "leaderboard", "visibility")
     list_filter = ("challenge_phase", "dataset_split", "leaderboard", "visibility")
-    search_fields = ("challenge_phase", "dataset_split", "leaderboard")
+    search_fields = ("challenge_phase__name", "dataset_split__name", "leaderboard__schema")
```

Comment:
I think we should also add `dataset_split__codename` in the search fields. Also, searching by `leaderboard_id` will make sense instead of `leaderboard_schema`. Can you please change it.

Reply:
@RishabhJain2018 searcing with the relation "__id" doesn't give any result. Should we do this?

Reply:
@RishabhJain2018 Please guide here. Should we add "leaderboard__id" or left it as it is in reference to the above comment! :-)

Reply:
@HargovindArora Sorry for replying late. I somehow missed this thread.
Now, coming back Yes, we should add `leaderboard__id` in the `search_fields` as when we will search with it, then it will return the list of all `challenge_phase_splits` that use the same leaderboard. 
For Example: If leaderboard id `101` is being used in `3` challenge phase splits then to list all those `challenge_phase_splits` we need to search with `id`. Please tell if you still have doubts.
On the contrary I didn't see any results with field `leaderboard__schema`. Can you please tell if I am missing something here ?